spec/factories/display_ads.rb

```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :display_ad do
     placement_area { "sidebar_left" }
-    body_markdown { "Hello _hey_ Hey hey" }
+    body_markdown { "Hello _hey_ Hey hey #{rand(1000000000)}" }
```

Comment:
While unlike, `rand` could still lead to collisions. Should we use a sequence instead?

```suggestion
    sequence(:body_markdown) { |n| "Hello _hey_ Hey hey #{n}" }
```